server/middleware/errorHandler.ts

```
@@ -0,0 +1,114 @@
+  }
+
+  // 開発環境でのみスタックトレースを含める
+  if (process.env.NODE_ENV === 'development') {
+    response.stack = error.stack
+  }
+
+  res.status(statusCode).json(response)
+}
+
+export const asyncHandler = (fn: Function) => {
+  return (req: Request, res: Response, next: NextFunction) => {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)